src/uct/ib/dc/dc_mlx5.h

```diff
@@ -119,8 +119,11 @@ typedef void (*uct_dc_dci_handle_failure_func_t)(uct_dc_mlx5_iface_t *iface,
                                                  ucs_status_t status);
 
 
+/**
+ * DCI interface operations.
+ */
 typedef struct uct_dc_dci_ops {
-    uct_dc_dci_handle_failure_func_t handle_failure;
+    uct_dc_dci_handle_failure_func_t handle_failure; /* error handling */
```

Comment:
callback for handling completion with error on the DCI

Reply:
reworded